packages/api/routes/apikeys/key/deleteKeyMiddleware.js

```diff
@@ -0,0 +1,34 @@
+const db_apikey = require("../../../lib/db.apikey");
+var validate = require("uuid-validate");
+
+// Return a function for deleting an API key passed as a parameter.
+module.exports = function deleteKeyMiddleware() {
+  return async (req, res, next) => {
+    const apikey = await db_apikey.attach();
+    const hashedKey = req.query.hashedKey ? req.query.hashedKey.trim() : req.query.hashedKey;
+
+    // Validate that the HashedKey confirms to uuid v4.
+    const isValid = validate(hashedKey, 4);
```

Comment:
This will always return false, because `hashedKey` is not a uuid.  `hashedKey` is a sha256 hash of the key.  This should be validating that the string is a 64 hex characters.